Python/flowgraph.c

```diff
@@ -1479,16 +1493,12 @@ apply_static_swaps(basicblock *block, int i)
 }
 
 static int
-optimize_basic_block(PyObject *const_cache, basicblock *bb, PyObject *consts)
+basicblock_fold_load_const(PyObject *const_cache, basicblock *bb, PyObject *consts)
```

Comment:
"fold" has many meanings in CS.
Maybe `fold_jumps`?

Reply:
It's not only jumps. This function also fold LOAD_CONST + POP_TOP/RETURN_VALUE/TO_BOOL etc

Reply:
I'll call it `optimize_load_const` instead of `fold_load_const`.